climpred/comparisons.py

```diff
@@ -278,3 +280,16 @@ def _m2r(ds, reference):
     reference = reference.isel(member=[0] * nMember)
     reference['member'] = forecast['member']
     return forecast, reference
+
+
+ALL_HINDCAST_COMPARISONS_DICT = {
```

Comment:
I think constants should be defined at the top of the file.

Reply:
Agreed

Reply:
when I move it up to the top of the file, I get errors because metrics/comparisons in the dict are not defined (yet). thats why I put them in the end.